src/renderer/components/dialogs/ForwardMessage/index.tsx

```diff
@@ -22,14 +22,15 @@ import styles from './styles.module.scss'
 const LIST_FLAGS = C.DC_GCL_FOR_FORWARDING | C.DC_GCL_NO_SPECIALS
 
 export default function ForwardMessage(props: {
-  message: T.Message
+  message: T.Message | number[]
```

Comment:
I think the API got a little more confusing now as `message` (singular) can now be a single message object or a list of message ids (plural).

How about:

```t
messages: T.Message[] | number[]
```

Ideally even giving `number[]` a type so it gets clearer what the number represents. Maybe we even have that type somewhere already.

```ts
type MessageId = number

// ...

messages: T.Message[] | MessageId[]
```

Or even better, just remove the object support and only accept IDs from now on:

```ts
messageIds: number[]
```

Reply:
I agree. But this is for another PR IMO.